format/spec.md

```diff
@@ -449,7 +454,7 @@ Partition field IDs must be reused if an existing partition spec contains an equ
 
 | Transform name    | Description                                                  | Source types                                                                                              | Result type |
 |-------------------|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|-------------|
-| **`identity`**    | Source value, unmodified                                     | Any                                                                                                       | Source type |
+| **`identity`**    | Source value, unmodified                                     | Any except `geometry` and `geography`                                                                     | Source type |
```

Comment:
I think we were also supposed to include Variant in the exceptions. @aihuaxu can you take a look?